### PR TITLE
RRP repo doc updates and cleanup

### PR DESCRIPTION
**Pull request by sjspielman**
**Stacked on #134**

This PR updates some items I noticed should happen when I was reviewing #134. 

There are a couple "bigger" changes I jumped the gun on a little bit, so tagging in @jashapiro to tell me if I went too far without discussing things first! Specifically, we really don't seem to use/need the `slides/*url` files, so I removed this directory and in the new README I indicate that slide links are available in the schedule directly. In addition, I renamed all the instructor notes to be in their own order of presentation.

For @Jen-OMalley, any feedback on README wording (there are 2 READMEs heres!) let me know!

### CODE REVIEW

**Comment by jashapiro** [comment]
I like the `slides` folder, so I would keep it (but update any URLs that need updating). I found it handy for going right to the slides when working on them. We should add in the data sharing as well, I expect then.

**Comment by sjspielman** [comment]
> I like the slides folder, so I would keep it (but update any URLs that need updating). I found it handy for going right to the slides when working on them. We should add in the data sharing as well, I expect then.

Thanks @jashapiro, reversions incoming!

One "motivation" I actually had here was keeping numbering consistent since [we added a slide in the middle](https://oklahoma.agclassroom.org/resources_food/songs/pancake_poem/)...and rather than changing all the numbers and maintaining those numbers over time I burned it all down....
Would you feel ok if I restored this directory _but_ didn't number the files in order? They'd be numbered in the README!

**Comment by jashapiro** [comment]
> Would you feel ok if I restored this directory _but_ didn't number the files in order? They'd be numbered in the README!

Personally, I would keep them numbered. But I don't feel particularly strongly about it.

File: workshop_materials/slides/03_data-sharing.url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[InternetShortcut]
+URL=https://docs.google.com/presentation/d/10NEkqHkAw9KBAbkNV-vtPwLFc9ZHksHEcCFty32efiE/edit?usp=sharing
```
Comment (sjspielman) [inline workshop_materials/slides/03_data-sharing.url:2]:
```suggestion
URL=https://docs.google.com/presentation/d/10NEkqHkAw9KBAbkNV-vtPwLFc9ZHksHEcCFty32efiE/edit?usp=sharing

```

**Comment by sjspielman** [comment]
@jashapiro Back to how it all was before after my gratuitous 🎢 but with updated numbers!
@Jen-OMalley, now it's definitely ready for you. In addition to docs to review, there are now also file _name changes_ for slide ordering and a new slide file to insert data sharing.

**Review by Jen-OMalley — APPROVED** [review]
All LGTM!

**Comment by sjspielman** [comment]
@Jen-OMalley I went through this and made sure everything is where it should be!

- Instructor notes numbering matches associated slide numbering
- All slide links work
- All READMEs contain the correct content

Give me a 👍 here if you agree that I got everything where it should be, and I'll go ahead and wrap this up at long long 🙃

**Comment by Jen-OMalley** [comment]
Thumbs up given!👍🏻 LGTM @sjspielman!

**Pull request merged**